apps/ledger-live-mobile/src/components/AccountCard.tsx

```diff
@@ -36,12 +40,16 @@ const AccountCard = ({
   ...props
 }: Props) => {
   const { colors } = useTheme();
+  if (!account) return null;
   const currency = getAccountCurrency(account);
   const unit = getAccountUnit(account);
   const tag =
-    account.derivationMode !== undefined &&
-    account.derivationMode !== null &&
-    getTagDerivationMode(currency as CryptoCurrency, account.derivationMode);
+    (account as Account)?.derivationMode !== undefined &&
```

Comment:
it would be better to check `account.type==="Account"` to not have to cast a AccountLike into Account as this isn't really safe – optionally we could also have a `getAccountTag(accountLike)`

Reply:
way better indeed 👍